sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/RateStreamProviderSuite.scala

```diff
@@ -78,6 +78,22 @@ class RateStreamProviderSuite extends StreamTest {
     )
   }
 
+  test("stream table API support") {
```

Comment:
can we move `DataStreamTableAPISuite` to a new file and move this test to there as well?

Reply:
Yes, done in 9004fba